libs/src/services/ethContracts/ethRewardsManagerClient.js

```diff
@@ -0,0 +1,35 @@
+const ContractClient = require('../contracts/ContractClient')
+
+class EthRewardsManagerClient extends ContractClient {
```

Comment:
there will need to be some steps to get this initialized i believe -- have you tried that yet? e.g. https://github.com/AudiusProject/audius-protocol/blob/master/libs/src/services/ethContracts/index.js#L75

Reply:
oh nice! thanks